package/yast2-testsuite.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Feb 26 11:12:58 UTC 2019 - José Iván López González <jlopez@suse.com>
```

Comment:
I thought that we should avoid to exceed the length of `----` line. You have a quite long name (like me :stuck_out_tongue_closed_eyes:)

Reply:
Well, as commented in IRC, I always keep that line just as `osc vc` generates it.